conf/touchpoint.CODE.conf

```diff
@@ -0,0 +1,34 @@
+# ***NO PRIVATE CREDENTIALS IN THIS FILE *** - use subscriptions-frontend in S3 for private data
+touchpoint.backend.environments {
+    CODE {
+        salesforce {
+            consumer {
+                key = ""
+                secret = ""
+            }
+            api {
+                url = "https://test.salesforce.com"
+                username = ""
+                password = ""
+                token = ""
+            }
+        }
+        zuora {
+            paymentDelayInDays = 14
+            paymentDelayGracePeriod = 2
+            api {
+                username = ""
+                password = ""
+            }
+        }
+        stripe {
+            api.key {
+                secret = ""
+                public = "pk_test_Qm3CGRdrV4WfGYCpm0sftR0f"
```

Comment:
is it ok to check this in?

Reply:
Yes that's a public key, the secret on is in private config